setup.py

```diff
@@ -1,106 +1,70 @@
+import sys
+from distutils import log
 from pathlib import Path
-from shutil import move, rmtree
-from subprocess import check_call
-from tempfile import mkdtemp
 
 from setuptools import find_packages, setup
 from setuptools.command.build_py import build_py
 from setuptools.command.develop import develop
-from setuptools.command.sdist import sdist
 
-try:
-    # OSX Homebrew fix: https://stackoverflow.com/a/53190037/1067833
-    from sys import _base_executable as executable
-except ImportError:
-    from sys import executable
 
-_EGG_REQ_PATH = Path(__file__).parent.joinpath(
-    'py3_validate_email.egg-info', 'requires.txt')
-_REQ_PATH = Path(__file__).parent.joinpath('requirements.txt')
-
-with open(_REQ_PATH if _REQ_PATH.exists() else _EGG_REQ_PATH) as fd:
-    _req_content = fd.readlines()
-_DEPENDENCIES = [x.strip() for x in _req_content if x.strip()]
-
-with open(Path(__file__).parent.joinpath('README.rst')) as fd:
-    _LONG_DESC = fd.read()
-
-
-def run_initial_updater():
+def run_initial_updater(path):
```

Comment:
```suggestion
def run_initial_updater(path: Path):
```

Reply:
Do you have a tool to check the completeness of type annotations, or are you just so attentive? I really try to pay attention but I seem to forget this at least in one place for each of my commits :-/

Reply:
One could use mypy for that, I don't because I couldn't set it up properly with Django. 

I use python-language-server with sublime, and that will show completions for function input parameters only if the type is specified there. So basically it's grown to be a habit for me.